app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  # Base controller for api namespace
+  class BaseController < ApplicationController
+    rescue_from ActiveRecord::RecordNotFound, with: :missing_record
+
+    def missing_record
```

Comment:
Does this give us a way to address [client/issue 9](https://github.com/midohiofoodbank/freshtrak-client/issues/9)?

Reply:
I thought about that too but I think you want a specific message about an invalid zip code versus a generic record not found which could be for another reason.